index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+		<link rel="stylesheet" href="style.css">
+		<link rel="preconnect" href="https://fonts.googleapis.com">
+		<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+		<link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@300&family=Roboto:wght@100&display=swap" rel="stylesheet">
+		
+
+</head>
+<body>
+	<header id="ToolBar">
+		<img src="Images\Logo Placeholder.svg" alt="Logo" id="logo-placeholder">
+
+		<a href="#" id="tool-Button">
+			<img src="Images\MenuIcon.svg" alt="Menu Icon">		
+		</a>
+	</header>
+
+		<main id="headline">
+			<header id="Introduction">
+				<div id="Name">
+					<p>
+						Hey There. <br>
+						I’m Hamid.
+					</p>
+
+				</div>
+				<div id="Status">
+					<p>I'm a Software Developer</p>
+				</div>
+			</header>
```

Comment:
- [ ] I would suggest you please remove this `header` tag as this tag is used for the page heading. I would suggest you please use the `section` tag as It represents a generic standalone section of the document